Eightshift/Sniffs/Security/ComponentsEscapeSniff.php

```diff
@@ -42,6 +42,16 @@ class ComponentsEscapeSniff extends EscapeOutputSniff
 	 */
 	public $overriddenClass = '';
 
+	/**
+	 * List of allowed methods that won't trigger the EscapeOutput error.
+	 *
+	 * @var true[]
```

Comment:
Note to self: Update this.

And add phpstan as a dev dependency for better type safety.